.bldr.toml

```diff
@@ -24,6 +24,7 @@ paths = [
   "api/config/shared/*",
   "api/interservice/pg_sidecar/*",
   "components/automate-gateway/api/authz/*",
+  "components/automate-gateway/api/iam/*",
```

Comment:
Just like we have to regenerate from proto file changes, when there are actual endpoint changes this may impact the bldr config, so `generate_bldr_config` regenerates this file.